rule.go

```diff
@@ -44,6 +48,33 @@ type ContainerOverride struct {
 	Environment map[string]string `yaml:"environment,omitempty"`
 }
 
+// NetworkConfiguration represents ECS network configuration
+type NetworkConfiguration struct {
+	AwsVpcConfiguration *AwsVpcConfiguration `yaml:"aws_vpc_configuration"`
+}
+
+// AwsVpcConfiguration represents AWS VPC configuration
+type AwsVpcConfiguration struct {
+	Subnets        []string `yaml:"subnets"`
+	SecurityGroups []string `yaml:"security_groups,omitempty"`
+	AssinPublicIP  string   `yaml:"assign_public_ip,omitempty"`
+}
+
+func (nc *NetworkConfiguration) ecsParameters() *cloudwatchevents.NetworkConfiguration {
+	awsVpcConf := &cloudwatchevents.AwsVpcConfiguration{
+		Subnets: aws.StringSlice(nc.AwsVpcConfiguration.Subnets),
```

Comment:
Subnets is required, the other params are optional.
https://docs.aws.amazon.com/sdk-for-go/api/service/ecs/#AwsVpcConfiguration